docs/conf.py

```diff
@@ -127,8 +127,12 @@
 html_favicon = "images/favicon.ico"
 html_last_updated_fmt = "%Y/%m/%d"
 html_context = {
-    "analytics_enabled": bool(os.getenv("QISKIT_ENABLE_ANALYTICS", ""))
-}  # enable segment analytics for qiskit.org/documentation
+    # Enable segment analytics for qiskit.org/documentation
+    "analytics_enabled": bool(os.getenv("QISKIT_ENABLE_ANALYTICS", "")),
+    "theme_announcement": "🎉 Qiskit is getting a new documentation experience on IBM Quantum!",
+    "announcement_url": "https://docs.quantum-computing.ibm.com/",
+    "announcement_url_text": "Check it out",
```

Comment:
This feels like a slightly weird coupling between the theme and the repository, but not a huge deal.

Minor: the split of "namespacing" one variable with `theme_` but not the others is a bit odd to me.

(Personally, I'd prefer we namespace everything Qiskit-specific with a `qiskit_` prefix, since we have to share the namespace with everything Sphinx-related, and reading the conf file, it's unclear to me now what's related to the theme and what's not, but that's a relatively minor issue and out of scope any way.)

Reply:
Yeah I agree. Too late to change now that it's been released though.